deploy/setup.sh

```diff
@@ -1,45 +1,11 @@
 #!/bin/bash
```

Comment:
You can remove this file completely and add the line
```
- import_playbook: plays/setup.yml
```
between lines 2 and 3 of playbook.yml file.